src/main/groovy/com/google/protobuf/gradle/GenerateProtoTask.groovy

```diff
@@ -330,12 +330,67 @@ public abstract class GenerateProtoTask extends DefaultTask {
   @Internal("Input captured by getAlternativePaths(), this is used to query alternative path by locator name.")
   abstract MapProperty<String, FileCollection> getLocatorToAlternativePathsMapping()
 
-  @InputFiles
-  @PathSensitive(PathSensitivity.NONE)
+  @Internal
+  abstract MapProperty<String, String> getLocatorToDependencyMapping()
+
+  @Internal
```

Comment:
This was only present for capturing input, so if it is internal it is unused. Delete?

Reply:
Deleting certainly wouldn't break anything here, but I wasn't sure if any build scripts may be referencing the property.

Reply:
Oh, I see. Probably highly unlikely, but yeah, I guess leave it.